SEC/Equities/EquityInstruments.rdf

```diff
@@ -628,7 +632,7 @@
 		<rdfs:subClassOf rdf:resource="&fibo-sec-eq-eq;PreferredShare"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fbc-fi-fi;hasMaturityDate"/>
+				<owl:onProperty rdf:resource="&fibo-fbc-dae-dbt;hasMaturityDate"/>
```

Comment:
would this allow a maturity date that was not an explicit date?

Reply:
I don't think so - the range of hasMaturityDate is ExplicitDate, so making it Date would violate our rules with respect to level crossing

Reply:
I was trying to understand the purpose/meaning of having an explicit qualified cardinality (onClass) when the cardinality is zero. Why not just remove the onClass triple?

Reply:
@rivettp So the logic here says that it cannot have a value for maturity date.  Are you suggesting that it should be an unqualified cardinality?  The onClass is strictly for completeness, and for consistency in visualization in some tools.